arduino/KnexElevator/DataAcquisitionTask.cpp

```diff
@@ -34,13 +34,9 @@
 #include "DataAcquisitionTask.h"
 
 
-// /////////////////////////////////////////////////////////////////////////////
-// Local Types
-// /////////////////////////////////////////////////////////////////////////////
-
-
 ////////////////////////////////////////////////////////////////////////////////
 DataAcquisitionTask::DataAcquisitionTask(void)
+    : ITask(__FUNCTION__, ITask::Defaults::Priority, 200)
```

Comment:
Bumped up the stack size for the DataAcquisition task after taking a look at the system task output. There were 0 bytes remaining in the stack watermark... it either blew out, or was very close to doing so. With this configuration, there are 93 bytes in the watermark.

Please see Issue #12 for some more details that will let us see this type of issue in the future.